source/agora/common/crypto/ECC.d

```diff
@@ -90,11 +91,29 @@ public struct Scalar
         crypto_core_ed25519_scalar_reduce(this.data[].ptr, param[].ptr);
     }
 
+    public enum PrintMode
```

Comment:
Needs `///` style comment. And also a description of what each mode does in the comment.

Reply:
Move this enum to `agora.common.Types` to avoid the duplication